Integrations/integration-HybridAnalysis.yml

```
@@ -630,6 +630,9 @@ script:
 
     //Quick Scan commands
     function quickScanUrl(scan_type, url, endpoint) {
+        url = url.replace('?', '%3F');
+        url = url.replace('=', '%3D');
+        url = url.replace('&', '%26');
         var body = 'scan_type=' + scan_type + '&url=' + url;
```

should `scan type` be encoded too?
```suggestion
        var body = 'scan_type=' + scan_type + '&url=' + encodeURIComponent(url);
```